third_party/py/abseil/absl/app.py

```diff
@@ -204,7 +204,7 @@ def register_and_parse_flags_with_usage(argv=None):
   return argv
 
 
-def _run_main(main, argv):
+def _run_main(main, argv, exit=True):
```

Comment:
Is there a better name we could use instead of `exit`? `exit` is a reserved keyword in some other programming languages (not python, to be clear), and I think we could do better here. The intent of this argument is to optionally allow the function to call `sys.exit()`, so maybe we could call it `exits_after_run`?

Reply:
Ah I forgot this change was in abseil, so disregard this comment for now. Changing absl would be a last resort option for me; see my other comment about possibly not using absl at all.